packages/shared/src/utils/debug/dbPerfMonitor.ts

```
@@ -303,6 +307,14 @@ function logLocalDbCall(method: string, table: string, params: any[]) {
   if (IS_ENABLED) {
     // eslint-disable-next-line no-param-reassign
     method = `localDb.${method}`;
+    // eslint-disable-next-line no-param-reassign
+    params = [...params, errorUtils.getCurrentCallStack()];
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)